nautobot/ipam/tests/test_api.py

```diff
@@ -1297,24 +1297,24 @@ def setUpTestData(cls):
         # make sure there are 4 locations without vlans 1 and 2 for the create_data below
         for i in range(4):
             cls.locations[i].vlans.set([])
-        locations_without_vlans = cls.locations.exclude(vlans__in=[cls.vlans[0], cls.vlans[1]])
+        locations_without_vlans = cls.locations.exclude(vlans__isnull=False)
```

Comment:
Perhaps:

```suggestion
        locations_without_vlans = cls.locations.filter(vlans__isnull=True)
```

Reply:
Are we guaranteed to have four locations with no associated VLANs? Or should we just explicitly create four such locations here to make the test more robust?

In either case, should the fixes in this file be backported to `develop` as well?

Reply:
Yeah we were using index 1-4 which are expecting 5 locations it should be 0-3. Took me a while to figure out the obvious lol. I will backported it to develop

Reply:
Ok - are we guaranteed to always have 4 suitable locations with the current implementation and factories?

Reply:
Yes definitely, we will have more than 10 locations with VLAN content_type. And we are using a for loop above to make sure that we have 4.